.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish Pipeline-ai 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
```

Comment:
Maybe this is obvious, but the push to real PyPI only happens if the commit ref is tag. If I push to GitHub, then create the tag and push that, will the workflow be re-run, triggering the 'real' release? I guess I'm wondering if "push" means only "commit pushes" or also "tag pushes".

Reply:
means any push onto a branch